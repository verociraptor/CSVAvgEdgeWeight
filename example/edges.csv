Edges,A,B,C,D
A,1,0,2,0
B,1,2,3,4
C,1,5,2,6
D,2,3,2,1
