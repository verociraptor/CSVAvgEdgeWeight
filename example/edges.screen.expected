Average Edge Weight: 2.1875
